docs/cloud/aws/lca1/customer-ssp/index.md

```diff
@@ -0,0 +1,97 @@
+# System Security Plan (SSP) Information
+
+Systems that fall under the [CU Data Classification](https://www.cu.edu/data-governance/resources-support/data-classification) of **Confidential Information** or
+**Highly Confidential Information** are required to provide a System Security Plan (SSP).
+An SSP is a document that outlines how an organization protects the security of a system.
+An SSP is a critical document that helps organizations meet compliance requirements.
+
+The [OIT IT Security](https://oit.colorado.edu/services/it-security) group is responsible for guiding and evaluating your SSP.
```

Comment:
I think they generally go by "OIT Security" not "OIT IT Security".

Reply:
Looking here: https://oit.colorado.edu/services/it-security it looks like they go by "IT Security".  How about `OIT's [IT Security](link)`?

Reply:
You pick the one that you think is the most right. Either way is fine.

Reply:
updated

Reply:
Updated all references to "IT Security" or OIT's "IT Security" to be consistent.